mediator/med_phases_aofluxes_mod.F90

```diff
@@ -870,11 +870,12 @@ subroutine med_aofluxes_init_xgrid(gcomp, aoflux_in, aoflux_out, rc)
          regridmethod=ESMF_REGRIDMETHOD_CONSERVE_2ND, rc=rc)
     if (chkerr(rc,__LINE__,u_FILE_u)) return
     if (trim(coupling_mode) == 'cesm') then
+       stp = 1
        call ESMF_FieldRegridStore(field_a, field_x, routehandle=rh_agrid2xgrid_bilinr, &
-            regridmethod=ESMF_REGRIDMETHOD_BILINEAR, dstMaskValues=(/0/), rc=rc)
+            regridmethod=ESMF_REGRIDMETHOD_BILINEAR, dstMaskValues=(/0/), srcTermProcessing=stp, rc=rc)
```

Comment:
These changes also fix the issue related to the xgrid in UFS end. We had issue with bilinear and each under UFS and also reproducibility issue when threading is enabled for xgrid specific RT. So, we make agrid default for it. Anyway, I am plaining to test this version of CMEPS to see it will help about those issues under UFS. I'll update about it.

Reply:
@uturuncoglu I just saw this comment. So this would allow us to set that test from agrid to xgrid if it works, correct?

Reply:
@DeniseWorthen Yes. It would be nice to test it. Do you want me to test it? I could but it could take time since I am trying to finalize my presentation for the CESM workshop but I could do it on Thursday. The exchange grid implementation was giving error for bilinear and patch interpolations. So, we have same specialization in CMEPS side due to it.

Reply:
@uturuncoglu There's no real rush. If you get a chance to test, that's great. I might be able to find time also but you'd need to let me know exactly how you were testing before, but next week would be fine.